docs/index.md

```diff
@@ -21,37 +21,49 @@ Development of *pydantic* is made possible by the following sponsors:
   <div>
     <a rel="sponsored" target="_blank" href="https://www.salesforce.com">
       <img src="./sponsor_logos/salesforce.png" alt="Salesforce" />
-      Salesforce
+      <span>Salesforce</span>
     </a>
   </div>
   <div>
     <a rel="sponsored" target="_blank" href="https://fastapi.tiangolo.com">
       <img src="./sponsor_logos/fastapi.png" alt="FastAPI" />
-      FastAPI
+      <span>FastAPI</span>
+    </a>
+  </div>
+  <div>
+    <a rel="sponsored" target="_blank" href="https://aws.amazon.com">
+      <img src="./sponsor_logos/aws.png" alt="AWS" />
+      <span>AWS</span>
+    </a>
+  </div>
+  <div>
+    <a rel="sponsored" target="_blank" href="https://explosion.ai">
+      <img src="./sponsor_logos/explosion_ai.png" alt="Explosion AI" />
+      <span>Explosion AI</span>
```

Comment:
Looks great otherwise, thanks! 🙏

```suggestion
      <img src="./sponsor_logos/explosion_ai.png" alt="Explosion" />
      <span>Explosion</span>
```